src/generators/essudpgen/ReadoutGeneratorBase.cpp

```diff
@@ -27,10 +30,11 @@ ReadoutGeneratorBase::ReadoutGeneratorBase() {
   app.add_option("-s, --pkt_throttle", Settings.PktThrottle,
                  "Extra usleep() after n packets");
   app.add_option("-y, --type", Settings.TypeOverride, "Detector type id");
-  app.add_option("-r, --rings", Settings.NFibers,
```

Comment:
@mortenjc 

Hi, I removed this setting since it's mentioned as obsolete since a while (since it's duplicated -f) now, and I made the -r option free. Few lines later I added -r as a character for randomize. I would make -m free for later use.

Let me know your opinion, but I checked our deployment and -r used only one place for freia so before the deployment of this change I have to change that to use -f intead.